client/state/action-types.js

```diff
@@ -564,6 +564,8 @@ export const MASTERBAR_TOGGLE_VISIBILITY = 'MASTERBAR_TOGGLE_VISIBILITY';
 export const MARKETING_CLICK_UPGRADE_NUDGE = 'MARKETING_CLICK_UPGRADE_NUDGE';
 export const MEDIA_DELETE = 'MEDIA_DELETE';
 export const MEDIA_ITEM_DELETE = 'MEDIA_ITEM_DELETE';
+export const ADD_GUTENFRAME_MEDIA_ACTION = 'ADD_GUTENFRAME_MEDIA_ACTION';
```

Comment:
It's generally a good idea to keep this list alphabetized. 

With that in mind, we often keep the actual action at the end of the name. Applying this rule to the naming, we'd end up with: `GUTENFRAME_MEDIA_ACTION_ADD` and `GUTENFRAME_MEDIA_ACTION_REMOVE`.